src/instances/HeadcountInstance.ts

```diff
@@ -128,6 +129,16 @@ export class HeadcountInstance {
     private static readonly DEFAULT_EMBED_COLOR: number = 16777215; //default to white
     private _embedColor: number;
 
+    // The headcount start time and expiration time
+    private _startTime: number;
+    private _expTime: number;
+    private static readonly DEFAULT_HEADCOUNT_DURATION: number = 60*60*1000; //1 hour in milliseconds
+
+    // Instance information for logging
+    private _instanceInfo: string;
+
+    // Time between panel updates in ms
+    private readonly _intervalDelay: number = 5000;
```

Comment:
If this isn't going to change between instances, make this `private static readonly ...`